test/packetforward/packetforward_upgrade_test.go

```diff
@@ -0,0 +1,117 @@
+package packetforward_test
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	v1 "github.com/celestiaorg/celestia-app/pkg/appconsts/v1"
+	v2 "github.com/celestiaorg/celestia-app/pkg/appconsts/v2"
+	"github.com/celestiaorg/celestia-app/test/util"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	packetforwardtypes "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v6/router/types"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	version "github.com/tendermint/tendermint/proto/tendermint/version"
+	dbm "github.com/tendermint/tm-db"
+)
+
+// TestPacketForwardMiddlewareAgainstAppUpgrades verifies that the PFM module's params are overridden during an
+// upgrade from v1 -> v2 and correctly set when upgrading to v2.
```

Comment:
[nit] the second half is redundant

```suggestion
// TestPacketForwardMiddlewareAgainstAppUpgrades verifies that the PFM module's params are overridden during an
// upgrade from v1 -> v2.
```